app/src/main/java/com/duckduckgo/app/notification/NotificationRegistrar.kt

```diff
@@ -71,11 +71,13 @@ class NotificationRegistrar @Inject constructor(
         )
     }
 
-    private val channels = (listOf(
-        ChannelType.FILE_DOWNLOADING,
-        ChannelType.FILE_DOWNLOADED,
-        ChannelType.TUTORIALS
-    ))
+    private val channels = (
+        listOf(
+            ChannelType.FILE_DOWNLOADING,
+            ChannelType.FILE_DOWNLOADED,
+            ChannelType.TUTORIALS
+        )
+        )
```

Comment:
This is so odd 🤷‍♂️

Reply:
interestingly enough, DDG.xml formats it the same way, just adding more continuation indent 🤷

Reply:
As it seems there's an extra parenthesis, I removed it. That seems what was causing this weird formatting in both ktlint and ddg.xml. See https://github.com/duckduckgo/Android/pull/1006/commits/2f32cdca635b20a82f5dfa818bd0491bb3cfda35